src/coreclr/debug/createdump/crashinfo.cpp

```diff
@@ -328,30 +328,31 @@ CrashInfo::EnumerateMemoryRegionsWithDAC(MINIDUMP_TYPE minidumpType)
     {
         TRACE("EnumerateMemoryRegionsWithDAC: Memory enumeration STARTED (%d %d)\n", m_enumMemoryPagesAdded, m_dataTargetPagesAdded);
 
-        // Since on MacOS all the RW regions will be added for heap dumps by createdump, the
-        // only thing differentiating a MiniDumpNormal and a MiniDumpWithPrivateReadWriteMemory
-        // is that the later uses the EnumMemoryRegions APIs. This is kind of expensive on larger
-        // applications (4 minutes, or even more), and this should already be in RW pages. Change
-        // the dump type to the faster normal one. This one already ensures necessary DAC globals,
-        // etc. without the costly assembly, module, class, type runtime data structures enumeration.
-        CLRDataEnumMemoryFlags flags = CLRDATA_ENUM_MEM_DEFAULT;
+        // CLRDATA_ENUM_MEM_HEAP2 skips the expensive (in both time and memory usage) enumeration of the
+        // low level data structures and adds all the loader allocator heaps instead. The below original
```

Comment:
```suggestion
        // low level data structures and adds all the loader allocator heaps instead. The older 'DbgEnableFastHeapDumps'
```
Since there are two env vars below it may help to name the one we are refering to.